x-pack/plugins/cloud_security_posture/common/schemas/csp_rule_template_metadata.ts

```diff
@@ -35,6 +35,7 @@ export const cspRuleTemplateMetadataSchemaV870 = rt.object({
     id: rt.string(),
     version: rt.string(),
     rule_number: rt.maybe(rt.string()),
+    posture_type: rt.maybe(rt.string()),
```

Comment:
`maybe` is used now because FTR runs with a custom version that doesn't have these yet.